events/api.py

```diff
@@ -1417,6 +1417,17 @@ class LanguageViewSet(JSONAPIViewMixin, viewsets.ReadOnlyModelViewSet):
     queryset = Language.objects.all()
     serializer_class = LanguageSerializer
 
+    def filter_queryset(self, queryset):
+        query_params = self.request.query_params
+
+        val = query_params.get("service_language", None)
+        if val is not None:
+            queryset = queryset.filter(
+                service_language=parse_bool(val, "service_language")
+            )
+
+        return queryset
+
```

Comment:
While this works and is consistent with many currently existing filters in LE, it also represents the kind of DIY style we want to get replaced by a more generic django-filter based solution, so ideally new filter code should use that. In this case I **think** just adding `filterset_fields` to the viewset should work (not sure at all, pls test 😄):
```suggestion
    filterset_fields = ("service_language",)
```

Reply:
Nice suggestion. Changed as suggested.